non-free.md

```diff
@@ -169,6 +170,13 @@
 - [Cosmos](https://cosmos-cloud.io/) - Cosmos is a self-hosted platform for running server applications securely and with built-in privacy features. It acts as a secure gateway to your application, as well as a server manager. ([Source Code](https://github.com/azukaar/cosmos-Server)) `⊘ Proprietary` `Docker/Go`
 
 
+### Resource Planning - Enterprise Resource Planning
+
+**[`^        back to top        ^`](##awesome-selfhosted---non-free-software)**
+
+- [YetiForce](https://yetiforce.com/en/) - Opensource CRM ERP with sales, marketing, accounting, HR, Support, Logistics and GDPR support. ([Demo](https://gitstable.yetiforce.com/), [Source Code](https://github.com/YetiForceCompany/YetiForceCRM)) `YetiForce Public License 5.0` `PHP`
```

Comment:
Rather than adding `YetiForce Public License 5.0`, Please change it to `⊘ Proprietary`